### PR TITLE
add admin job to delete a dynamodb row using ci

### PR DESCRIPTION
**Pull request by TheDogfather17**
Added an admin job that can take correlation Id and data product as an input. It can then delete that row in the pipeline metadata table to enable fresh cluster runs.

### CODE REVIEW

File: ci/jobs/preprod/admin/clear_dynamodb_row.yml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+jobs:
+  - name: preprod-clear-dynamodb-row
+    plan:
+      - .: (( inject meta.plan.clear-dynamodb-row ))
+        config:
+          params:
+            AWS_ROLE_ARN: arn:aws:iam::((aws_account.development)):role/ci
```
Comment (steveburton4) [inline ci/jobs/preprod/admin/clear_dynamodb_row.yml:7]:
```suggestion
            AWS_ROLE_ARN: arn:aws:iam::((aws_account.preprod)):role/ci
```

File: ci/jobs/preprod/admin/clear_dynamodb_row.yml [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+            AWS_ACC: ((aws_account.development))
```
Comment (steveburton4) [inline ci/jobs/preprod/admin/clear_dynamodb_row.yml:8]:
```suggestion
            AWS_ACC: ((aws_account.preprod))
```

File: ci/meta.yml [hunk 3]
```diff
@@ -439,3 +439,39 @@ meta:
...
+      config:
+        platform: linux
+        image_resource:
+          type: docker-image
+          source:
+            repository: ((dataworks.docker_awscli_repository))
+            version: ((dataworks.docker_awscli_version))
+            tag: ((dataworks.docker_awscli_version))
+        params:
+          AWS_ROLE_ARN: arn:aws:iam::((aws_account.management)):role/ci
```
Comment (steveburton4) [inline ci/meta.yml:454]: Please remove this default line

**Comment by steveburton4** [comment]
Can you add to the README please for what this does and how to use it. We should do this for all utility jobs we add (they are missing for now).

**Review by steveburton4 — CHANGES_REQUESTED** [review]

File: ci/meta.yml (hunk 3 above)
Comment (TheDogfather17) [reply ci/meta.yml]: that's done

**Review by steveburton4 — COMMENTED** [review]
README change still needed

**Review by ChrisScottThomas — APPROVED** [review]
LGTM

**Review by steveburton4 — APPROVED** [review]

**Pull request merged**